src/db.rs

```diff
@@ -210,6 +210,7 @@ impl Database {
             if self.handle_set(&stmt)? {
                 continue;
             }
+            let stmt_copy = stmt.clone();
```

Comment:
Could you avoid copy by moving the `header_values` up to here?

Reply:
ok. let me try